processingchain/src/test/groovy/org/orbisgis/orbisprocess/geoclimate/processingchain/ProcessingChainBDTopoTest.groovy

```diff
@@ -240,16 +263,27 @@ class ProcessingChainBDTopoTest extends ChainProcessAbstractTest{
                 abstractTables.outputHydro, saveResults, svfSimplified, indicatorUse,  prefixName)
     }
 
-    //@Test
+    // Test the workflow on the commune INSEE 01306 only for TEB in order to verify that only RSU_INDICATORS and BUILDING_INDICATORS are saved
+    @Test
+    @DisabledIfSystemProperty(named = "data.bd.topo", matches = "false")
     void testBDTOPO_V2Workflow() {
+        String inseeCode = "01306"
+        H2GIS datasource = loadFiles(inseeCode)
+
         String directory ="./target/geoclimate_chain"
         File dirFile = new File(directory)
+        File inpFolder = new File(ProcessingChainBDTopoTest.class.getResource("bdtopofolder").toURI())
         dirFile.delete()
         dirFile.mkdir()
-        H2GIS datasource = H2GIS.open(dirFile.absolutePath+File.separator+"geoclimate_chain_db;AUTO_SERVER=TRUE")
-        IProcess process = ProcessingChain.Workflow.BDTOPO_V2()
-        assertTrue(process.execute(datasource: datasource,
-                inputFolder: "./target/bdtopofolder/",
-                outputFolder :"./target/geoclimate_chain/"))
+        IProcess processBDTopo = ProcessingChain.Workflow.BDTOPO_V2()
+        assertTrue(processBDTopo.execute(datasource: datasource,
+                inputFolder: inpFolder.getAbsolutePath().toString(),
+                outputFolder :"./target/geoclimate_chain/",
+                indicatorUse: ["TEB"]))
+        String baseNamePathAndFileOut = processBDTopo.getResults().outputFolder + File.separator + "zone_" + inseeCode + "_"
```

Comment:
You should assert that `processBDTopo.getResults().outputFolder` is not null

Reply:
just assert this or add a condition to the initialisation of the variable and to the following assertions ?

Reply:
Just an assertion